components/layout/navbar.tsx

```diff
@@ -32,15 +32,23 @@ const Navbar: NextPage = () => {
 
   return (
     <>
-      <div className={styles.container}>
+    <div className={styles.container}>
+        <Avatar 
+          className={styles.avatar}
+          src='/images/cotejoAppUI.png'
```

Comment:
Try to save the file with the same standar than the others.
```suggestion
          src='/images/cotejo-app-ui.png'
```

Reply:
Ready